misc/python/materialize/cli/bazel.py

```diff
@@ -16,6 +16,10 @@
 
 from materialize import MZ_ROOT, ui
 
+# Path where we put the current revision of the repo that we can side channel
+# into Bazel.
+MZ_GIT_HASH_FILE = "/tmp/mz_git_hash.txt"
```

Comment:
What makes it safe to always write to this file in `/tmp`? Does Bazel make `/tmp` hermetic? Not that this is a huge risk, but if you have two worktrees of Materialize and you're building simultaneously, it seems like they'll compete to write to this file? 

Is there a way to write to a file in the current repo instead? That seems a bit less worrying to me.

Reply:
Could we use an env var instead? If the build is allowed to read random files maybe we can just read a special env var?

Reply:
Unfortunately it's not safe, two different Materialize worktrees building Bazel at the same time will totally compete with each other. The issue is Bazel doesn't expose the path to the current repo in a way that wouldn't invalidate the cache. Same with environment variables, anything we try passing into for the build process of a `rust_binary` or `rust_library` is tracked as well.

I'll ask around and see if I can find a better solution, but going to merge as-is for now!

Reply:
I'm building in multiple repos at once all the time. I guess we also can't use something like the PID in the path?